transform/mattermost-analytics/models/staging/mm_calls_test_go/stg_mm_calls_test_go__tracks.sql

```diff
@@ -2,7 +2,7 @@ SELECT
     id AS event_id
     , event_text AS event_name
     , event AS event_table
-    , actual_user_id AS user_id
+    , coalesce(actual_user_id, participant_id) AS user_id
```

Comment:
Coalescing user id with participant id makes sense only if it's the same value. A more robust design would create a lookup table of user id and possible participant ids.

Reply:
this model is from a `base` model that unions individual event tables. The `user_id` is `actual_user_id` in some tables and `participant_id` in the other. There are no tables where it contains both fields. Because of the union we are missing the `user_id` where it is `participant_id`.

Reply:
IMO, a look up table will make things really complex, I validated the logic against the prod staging table and the CI/CD table, and it works as expected.

Reply:
Still, if `user_id` and `participant_id` have different semantics, it should be reflected, even in downstream models. Otherwise it can become easy to introduce errors downstream.